changelog/unreleased/configurable-max-lock-cycles.md

```diff
@@ -0,0 +1,6 @@
+Enhancement: Make max lock cycles configurable
+
+When a file is locked the flock library will retry a given amount of times (with a increasing sleep time inbetween each round)
+Until now the max amount of such rounds was hardcoded to `10`. Now it is configurable, falling back to a default of `25`
+
+https://github.com/owncloud/ocis/pull/4959
```

Comment:
@kobergj this changelog entry is broken. We need the reva pull request in the first line of the link list because it provides the primary issue id which needs to resolve inside the repo.